src/develop/pixelpipe_cache.c

```diff
@@ -49,6 +49,7 @@ gboolean dt_dev_pixelpipe_cache_init(
   cache->hash = (uint64_t *)calloc(entries, sizeof(uint64_t));
   cache->used = (int32_t *)calloc(entries, sizeof(int32_t));
   cache->modname = (char **)calloc(entries, sizeof(char *));
+  cache->ioporder = (int32_t *)calloc(entries, sizeof(int32_t));
```

Comment:
Looks like it could be an array of structs, no? Not a biggie, of course.

Reply:
yes. maybe later :-)